programs/marginfi/tests/bank_variable_oracle_staleness.rs

```diff
@@ -41,11 +44,11 @@ async fn bank_oracle_staleness_test() -> anyhow::Result<()> {
     let borrower_token_account_f_sol = test_f.sol_mint.create_token_account_and_mint_to(0).await;
 
     borrower_mfi_account_f
-        .try_bank_deposit(borrower_token_account_f_usdc.key, usdc_bank, 1_000)
+        .try_bank_deposit(borrower_token_account_f_usdc.key, usdc_bank, 500)
         .await?;
 
     borrower_mfi_account_f
-        .try_bank_deposit(borrower_token_account_f_sol_eq.key, sol_eq_bank, 1_000)
+        .try_bank_deposit(borrower_token_account_f_sol_eq.key, sol_eq_bank, 500)
         .await?;
```

Comment:
Is there a reason why these changed to 500 from 1000 that would change test behavior? If no reason, ignore this comment.

Reply:
So I believe the intent of this test is to check that a stale oracle can a stale bank deposit (next to a non-stale one) can make the difference between having enough collat for a borrow/withdraw and not.

So since here the test borrow is worth $990 (test suite sol price of $10), having a USDC deposit worth $1000 would alone be enough to back that borrow, and would fail the first error assertion. Updated amounts make each deposit bank account for ~ half of the required collat for that SOL borrow.